src/components/BooksForm.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const BooksForm = () => {
+  const categories = [
+    'Action',
+    'Biography',
+    'History',
+    'Horror',
+    'Kids',
+    'Learning',
+    'Sci-Fi'];
+
+  return (
+    <form>
+      <input id="title" />
+      <select>
+        {
+          categories.map(cat => (
+            <option key={cat}>
+              {cat}
+            </option>
+          ))
+        }
+      </select>
+    </form>
+  );
+};
+
+export default BooksForm;
```

Comment:
- `BooksForm` component should be placed in `src/containers` directory.
- it should have a `state` with `title` and `category`
- it renders form with `title` input, `categories` select box and `submit` see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Form/RenderForm.png) from pathwright